data_layer/aptos/aptos.go

```diff
@@ -71,6 +71,9 @@ func NewAptos(root string) *Aptos {
 			panic(fmt.Errorf("DA network configuration not found for: %s", aptConfig.Network))
```

Comment:
even though this was there already, change this to returning an error and handle it